exporter/awsemfexporter/README.md

```diff
@@ -25,6 +25,7 @@ The following exporter configuration parameters are supported.
 | `max_retries`     | Maximum number of retries before abandoning an attempt to post data.   |    1    |
 | `dimension_rollup_option`| DimensionRollupOption is the option for metrics dimension rollup. Three options are available. |"ZeroAndSingleDimensionRollup" (Enable both zero dimension rollup and single dimension rollup)| 
 | `resource_to_telemetry_conversion` | "resource_to_telemetry_conversion" is the option for converting resource attributes to telemetry attributes. It has only one config onption- `enabled`. For metrics, if `enabled=true`, all the resource attributes will be converted to metric labels by default. See `Resource Attributes to Metric Labels` section below for examples. | `enabled=false` | 
+| `run_in_lambda` | "run_in_lambda" is the option to support AWS Lambda by directing EMF exporter output to stdout, which will then be sent to CWLogs by Lambda. | `false` | 
```

Comment:
Better to describe meaning, not condition. So not `run_in_lambda`, just something like `output_to_stdout`. Or what do you think about `output_file` and have it support special syntax for stdout similar to some tools, but could also be used for normal system file?

Reply:
Thanks. That sounds better. How about `output_destination`? In that case, I would like to keep `cloudwatch` as a default value for this and provide another option as `stdout`?

Reply:
I think output_destination is fine too unless it seems useful to have the ability to specify a file, which could be stdout too - I guess it's a good feature?

Reply:
I used `output_destination` in the config. It currently supports two options: `CloudWatch` and `stdout`. We can further enhance it to support file path in the future so that EMF exporter can direct its output to specific file.